### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by uwcdc**
Please review @lsetiawan @carlosgjs

### CODE REVIEW

File: python-project-template/.github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Before submitting your pull request to the repo, please ensure that you have completed the following criteria:
+<!-- 
+Thank you for your contribution to the repo :smile:
```
Comment (lsetiawan) [inline python-project-template/.github/pull_request_template.md:3]:
```suggestion
Thank you for your contribution to the repo :)
```

File: python-project-template/.github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Before submitting your pull request to the repo, please ensure that you have completed the following criteria:
```
[the remark this answers is not in the document]
Comment (uwcdc) [reply python-project-template/.github/pull_request_template.md:1]: What language do you suggest for instructions outside the comment section?

**Comment by uwcdc** [comment]
Let me know what you think about the changes @lsetiawan and @carlosgjs

**Review by carlosgjs — APPROVED** [review]
LGTM!

**Comment by lsetiawan** [comment]
@uwcdc I thought we discussed in slack to keep the `main` branch clean and that once PR is approved in our fork, another PR should be made to the source repo? I'm confused now.

**Pull request merged**